detox/test/e2e/16.location.test.js

```diff
@@ -35,4 +35,15 @@ describe(':ios: location', () => {
     await expect(element(by.text('Latitude: 20.1'))).toBeVisible();
     await expect(element(by.text('Longitude: 20.2'))).toBeVisible();
   });
+
+  if(':android: Should receive location (20,20)', async () => {
+    await device.relaunchApp({ permissions: { location: 'always' } });
```

Comment:
Can we add a test for `location: never` or it doesn't work this way?

Reply:
I'll need to double check, but considering if you've added the permission in the manifest, you'll never have a never..

However support for permissions properly on Android would be another wonderful thing.

Reply:
We can use `adb shell pm grant/revoke` to grant or revoke permissions on the device, so I might look at working on another PR to do that, just have to match the permissions we support on iOS to the equivalent on Android, and then again only the ones that are deemed dangerous permissions (and will ask the user).

Reply:
We currently do not support runtime permissions on Android, IIRC we just adb install with -g flag, this seems unnecessary.
We strive to make the API as cross platform as we can, location seems fall into the "same API" category, therefore, let's use the same exact tests for both platforms.